src/roiextractors/extraction_tools.py

```diff
@@ -487,9 +631,25 @@ def animate_func(i, imaging, im, ax):
 
 
 def check_keys(dict):
-    """
-    checks if entries in dictionary are mat-objects. If yes
-    todict is called to change them to nested dictionaries
+    """Check keys of dictionary for mat-objects.
+
+    Checks if entries in dictionary are mat-objects. If yes
+    todict is called to change them to nested dictionaries.
+
+    Parameters
+    ----------
+    dict: dict
+        Dictionary to check.
+
+    Returns
+    -------
+    dict: dict
+        Dictionary with mat-objects converted to nested dictionaries.
```

Comment:
I notice with these a slight inconsistency; sometimes a one-line description of a field ends in a period (forms a complete sentence) but other times it does not, such as https://github.com/catalystneuro/roiextractors/pull/235/files#diff-9db21a3b993bb8f4a57b59186e0b23167702413272b3f7aa302a7bf5d0414d8bR46

Reply:
Similar to https://github.com/catalystneuro/roiextractors/pull/235#discussion_r1317709879, this seems to mostly be an inconsistency with some of the earlier docstrings, the more recent all end in periods

Reply:
Looks like all [numpydoc examples](https://numpydoc.readthedocs.io/en/latest/format.html) of parameter/return descriptions end in periods.  But pydocstyle doesn't check this (unlike the first line which it checks for a period).

Reply:
From the conversation on https://github.com/catalystneuro/roiextractors/pull/235#discussion_r1317784019, is it useful at all in this case to give the return variable the same name as its built-in type?

Reply:
I wasn't too careful about it in this PR tbh.

Reply:
Of course not.  But that's a problem with the code -- not the docstring.  It's questionable to overwrite the name `dict` at all since it's a built-in, never mind the fact that it is not descriptive at all really.

Reply:
This sort of stuff could be part of a big refactor for clarity/readability, but I would want some clear actionable objectives since readability can be somewhat subjective.

Reply:
>  It's questionable to overwrite the name dict at all since it's a built-in

A good point


> This sort of stuff could be part of a big refactor for clarity/readability, but I would want some specific actionable objectives since readability can be somewhat subjective.

Can be many small refactors over time, starting with things like not overriding built-in names, and using full words to describe an object

Reply:
See #238.